2996_reflection/reflection.md

```diff
@@ -4211,21 +4284,20 @@ Modify (and clean up) the definition of _immediate-escalating_ in paragraph 18 t
 
 :::
 
-Add a new paragraphs prior to the definition of _manifestly constant-evaluated_ ([expr.const]{.sref}/21), and renumber accordingly:
+Add a new paragraph prior to the definition of _manifestly constant-evaluated_ ([expr.const]{.sref}/21), and renumber accordingly:
 
 ::: std
 ::: addu
 
-[21pre]{.pnum} A non-dependent expression or conversion is _plainly constant-evaluated_ if it is not in a complete-class context ([class.mem.general]{.sref}) and is either
-
-* [21pre.#]{.pnum} the `$constant-expression$` of a `$static_assert-declaration$` ([dcl.pre]{.sref}), or
-* [21pre.#]{.pnum} an initializer of a `constexpr` ([dcl.constexpr]{.sref}) or `constinit` ([dcl.constinit]{.sref}) variable.
+[21pre]{.pnum} A non-dependent expression or conversion is _plainly constant-evaluated_ if it an initializer of a `constexpr` ([dcl.constexpr]{.sref}) or `constinit` ([dcl.constinit]{.sref}) variable that is not in a complete-class context ([class.mem.general]{.sref}).
 
 [The evaluation of a plainly constant-evaluated expression `$E$` can produce injected declarations (see below) and happens exactly once ([lex.phases]{.sref}). Any such declarations are reachable from a point that follows immediately after `$E$`.]{.note}
 
 :::
 :::
 
+[We also intend for the "evaluating expression" of a consteval block to be plainly constant-evaluated, but this construct is introduced by a separate follow-on paper ([@P3289R0]).]{.draftnote}
```

Comment:
Will have to add a reference to this too, but definitely refer to the latest revision:
```suggestion
[We also intend for the "evaluating expression" of a consteval block to be plainly constant-evaluated, but this construct is introduced by a separate follow-on paper ([@P3289R1]).]{.draftnote}
```

Reply:
Done.